### DISCUSSION TITLE
Not syncronizing

### DISCUSSION THREAD

**Discussion by santom1977** [Chia-Network/chia-blockchain#8372] [Support]
Guys I need some help, please.

I am running chia on windows and currently I am having a problem with syncronization.

![Untitled](https://user-images.githubusercontent.com/90283782/132403267-04de4bcd-3201-4d51-b4dc-4ab9f65051ea.png)

do you know how to resolve this problem?

**Comment by jack60612** [comment]
> Guys I need some help, please.
> 
> I am running chia on windows and currently I am having a problem with syncronization.
> 
> ![Untitled](https://user-images.githubusercontent.com/90283782/132403267-04de4bcd-3201-4d51-b4dc-4ab9f65051ea.png)
> 
> do you know how to resolve this problem?

Have you tried restarting your computer?

**Comment by santom1977** [comment]
yep and nothing

**Comment by BrandtH22** [comment]
@santom1977 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
